### DISCUSSION TITLE
Update Documentation Storage Backends

### DISCUSSION THREAD

**Discussion by pocki** [jaegertracing/jaeger#4307] [Ideas]
Can you please update the documentations about the storage backends?
It seems that there are now lot of different backends supported (#638) but I can't find any documentation how to configure and use it.

**Comment by yurishkuro** [comment]
Configuration of officially supported backends is documented already: https://www.jaegertracing.io/docs/1.43/deployment/#span-storage-backends. For unofficial backends you have to look into their own documentation / repositories.

**Reply by pocki** [reply]
Thanks for the update. I thought, there are more official backends now. Some of them have still poor to no documentation
